src/ComputeSharp.D2D1/Shaders/Interop/D2D1PropertyBinding.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace ComputeSharp.D2D1.Interop;
+
+/// <summary>
+/// A model representing a property binding for an effect.
+/// </summary>
+public unsafe readonly struct D2D1PropertyBinding
+{
+    /// <summary>
+    /// Creates a new <see cref="D2D1PropertyBinding"/> instance with the specified parameters.
+    /// </summary>
+    /// <param name="name">The property name.</param>
+    /// <param name="getter">A pointer to the property getter callback</param>
+    /// <param name="setter">A pointer to the property setter callback</param>
+    internal D2D1PropertyBinding(string name, nint getter, nint setter)
```

Comment:
why not just take `void*` instead of `nint`?

Reply:
Fixed 🙂